slackroll

```diff
@@ -455,17 +455,34 @@ def get_blacklist(): # Get blacklist expression list
     return []
 
 def get_blacklist_re(): # Almost the same, but already converted to regular expressions
+    regex_list = []
```

Comment:
While reviewing this PR, I realized the empty string is a valid regular expression that always matches any string. I think we can take advantage of that to simplify the code a bit more and avoid the few if/else blocks. Let's suppose we have the following function:

    def split_blacklist_str(blacklist_str):
        if '@' not in blacklist_str:
            return (blacklist_str, '')
        return tuple(blacklist_str.split('@', 1))

You can then use that to get the pkg_regex and the url_regex inconditionally, and append the tuple without any inconvenience. If the URL part is empty, the empty regular expression will match every URL.

Reply:
Ok, I've added this. Simplifies the rest :+1: